src/app/devices/deviceContent/components/digitalTwin/digitalTwinContentContainer.tsx

```diff
@@ -6,11 +6,12 @@ import { Dispatch } from 'redux';
 import { connect } from 'react-redux';
 import { AnyAction } from 'typescript-fsa';
 import { DigitalTwinContentDispatchProps, DigitalTwinContent } from './digitalTwinContent';
-import { getModelDefinitionAction } from '../../actions';
+import { getModelDefinitionAction, getDigitalTwinInterfacePropertiesAction } from '../../actions';
```

Comment:
you don't need this file. Make `DigitalTwinContent` a FC. it's a small component